lib/mix/lib/mix/exceptions.ex

```diff
@@ -9,8 +9,10 @@ defmodule Mix.NoTaskError do
   defp msg(task) do
     msg = "The task #{task} could not be found"
     case did_you_mean(task) do
-      nil -> msg
-      similar -> msg <> ". Did you mean '#{similar}'?"
+      {similar, score} when score > 0.79 ->
```

Comment:
I think 0.8 is acceptable. I have used jarowinkler in a project and for people and places we used around 0.95 or 0.9.